source/slang/slang-check-conversion.cpp

```diff
@@ -753,6 +753,18 @@ namespace Slang
             return true;
         }
 
+        // nullptr_t can be cast into any pointer type.
+        if (as<NullPtrType>(fromType) && as<PtrType>(toType))
+        {
+            if (outCost)
+            {
+                *outCost = kConversionCost_None;
```

Comment:
Not very important, but: making the conversion cost `None` here means that we can't detect an overload where the parameter has type `NullPtr` as better than an overload with `Ptr<T>`, even though we should seemingly clearly prefer the former (like how `nullptr_t` works with overloading in C++).

This isn't very important because I'd like to change how we handle `null` in the long run so that we don't have this weird typing behavior.